spec/support/features/session_helpers.rb

```diff
@@ -0,0 +1,12 @@
+module Features
+  module SessionHelpers
+    def sign_in(who = :user)
+      user = who.is_a?(User) ? who : FactoryGirl.build(:user).tap(&:save!)
+      visit new_user_session_path
+      fill_in 'Email', with: user.email
+      fill_in 'Password', with: user.password
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.